py-polars/polars/expr/string.py

```diff
@@ -579,6 +579,82 @@ def strip(self, characters: str | None = None) -> Expr:
         """
         return wrap_expr(self._pyexpr.str_strip(characters))
 
+    def strip_prefix(self, prefix: str) -> Expr:
+        r"""
+        Remove prefix.
+
+        Parameters
+        ----------
+        prefix
+            The prefix to be removed. The prefix will be removed exactly once from the
+            string if found.
+
+        Examples
+        --------
+        >>> df = pl.DataFrame({"baz": ["foo:bar", "foofoo:bar"]})
+        >>> df
+        shape: (2, 1)
+        ┌────────────┐
+        │ baz        │
+        │ ---        │
+        │ str        │
+        ╞════════════╡
+        │ foo:bar    │
+        │ foofoo:bar │
+        └────────────┘
+
+        >>> df.select(pl.col("baz").str.strip_prefix("foo"))
+        shape: (2, 1)
+        ┌─────────┐
+        │ baz     │
+        │ ---     │
+        │ str     │
+        ╞═════════╡
+        │ :bar    │
+        │ foo:bar │
+        └─────────┘
+
+        """
+        return wrap_expr(self._pyexpr.str_strip_prefix(prefix))
+
+    def strip_suffix(self, suffix: str) -> Expr:
+        r"""
```

Comment:
I don't think the docstring needs the `r` prefix here?

```suggestion
        """
```